Dockerfile

```diff
@@ -20,7 +20,8 @@ USER root
 ENV CHEFDK_VERSION=3.0.36
 
 RUN apt-get install -y \
-        ruby && \
+    ruby ruby-dev && \
```

Comment:
This is minor, but since these are arguments to `apt-get install`, can we indent them another level?

```
RUN apt-get install -y \
      ruby \
      ruby-dev && \
    gem install bunder &&
```

Reply:
Updated